clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerMembershipManagerTest.java

```diff
@@ -768,26 +768,26 @@ public void testDelayedReconciliationResultAppliedWhenTargetChangedWithNewAssign
         String topic2 = "topic2";
         final TopicIdPartition topicId2Partition0 = new TopicIdPartition(topicId2, new TopicPartition(topic2, 0));
 
-        MembershipManagerImpl membershipManager = createMembershipManagerJoiningGroup();
+        ConsumerMembershipManager membershipManager = createMembershipManagerJoiningGroup();
         when(subscriptionState.hasAutoAssignedPartitions()).thenReturn(true);
         when(metadata.topicNames()).thenReturn(
-            mkMap(
-                mkEntry(topicId1, topic1),
-                mkEntry(topicId2, topic2)
-            )
+                mkMap(
+                        mkEntry(topicId1, topic1),
+                        mkEntry(topicId2, topic2)
+                )
         );
 
         // Receive assignment with only topic1-0, getting stuck during commit.
         final CompletableFuture<Void> commitFuture = mockNewAssignmentAndRevocationStuckOnCommit(membershipManager, topicId1,
-            topic1, Collections.singletonList(0), false);
+                topic1, Collections.singletonList(0), false);
 
         // New assignment adding a new topic2-0 (not in metadata).
         // No reconciliation triggered, because another reconciliation is in progress.
         Map<Uuid, SortedSet<Integer>> newAssignment =
-            mkMap(
-                mkEntry(topicId1, mkSortedSet(0)),
-                mkEntry(topicId2, mkSortedSet(0))
-            );
+                mkMap(
+                        mkEntry(topicId1, mkSortedSet(0)),
+                        mkEntry(topicId2, mkSortedSet(0))
+                );
```

Comment:
Should we remain the indent style?

Reply:
Yes. Thanks for catching it. When I copied the code with the clipboard, I think it reformatted. Entirely accidental.